acceptance-tests/apps/postgresqlapp/internal/app/app.go

```diff
@@ -1,33 +1,30 @@
 package app
 
 import (
-	"context"
 	"fmt"
 	"log"
 	"net/http"
+	"postgresqlapp/internal/connector"
 	"regexp"
 
-	"github.com/gorilla/mux"
+	"github.com/go-chi/chi/v5"
 	_ "github.com/jackc/pgx/v4/stdlib"
-
-	"postgresqlapp/internal/connector"
 )
 
 const (
 	tableName     = "test"
 	keyColumn     = "keyname"
 	valueColumn   = "valuedata"
 	tlsQueryParam = "tls"
-	schemaKey     = "schema"
 )
 
-func App(conn *connector.Connector) *mux.Router {
-	r := mux.NewRouter()
-	r.HandleFunc("/", aliveness).Methods(http.MethodHead, http.MethodGet)
-	r.Handle("/{schema}", checkSchemaMiddleware(http.HandlerFunc(handleCreateSchema(conn)))).Methods(http.MethodPut)
-	r.Handle("/{schema}", checkSchemaMiddleware(http.HandlerFunc(handleDropSchema(conn)))).Methods(http.MethodDelete)
-	r.Handle("/{schema}/{key}", checkSchemaMiddleware(http.HandlerFunc(handleSet(conn)))).Methods(http.MethodPut)
-	r.Handle("/{schema}/{key}", checkSchemaMiddleware(http.HandlerFunc(handleGet(conn)))).Methods(http.MethodGet)
+func App(conn *connector.Connector) http.Handler {
+	r := chi.NewRouter()
+	r.Head("/", aliveness)
```

Comment:
we are missing Get for "/" as well, although I am not sure if it is really used. Feel free to ignore

Reply:
As you say, it's never used, and didn't seem worth adding an extra line for.